enable_gps: True
gps_topic: "/alpha_rise/gps/fix"

enable_battery: True
battery_topic: "/battery"

enable_depth: True
depth_topic: "/alpha_rise/pressure"

enable_odom: True
odometry: "/alpha_rise/odometry/filtered/local"